install.sh

```diff
@@ -60,9 +60,18 @@ yarn_verify_integrity() {
   fi
 
   printf "$cyan> Verifying integrity...$reset\n"
+
+  # Check whether a gpg-agent process already exists for this session
+  existing_gpg_agent=$(pgrep gpg-agent || echo)
```

Comment:
This is the best solution I could come up so far; essentially to use what is hopefully a relatively cross-platform command to determine whether a `gpg-agent` process already exists before key import is (potentially) attempted.  If previous agents did *not* exist, we can go ahead and remove any which were created.

`pgrep` has been available on Mac OS X since release 10.8 (released July 2012), and is AFAIK available by default in most Linux distributions.